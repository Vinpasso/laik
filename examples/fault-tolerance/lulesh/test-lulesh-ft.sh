#!/bin/sh

export OMP_NUM_THREADS=1
EXECUTABLE="./lulesh"
mpirun --oversubscribe -n 8 "$EXECUTABLE" --failureCheckFrequency 5 --checkpointFrequency 10 --redundancyCount 1 --rotationDistance 1 --plannedFailure 1 32 -s 10 -i 50 -p -repart 1 > test-lulesh-ft.out
cmp test-lulesh-ft.out "$(dirname -- "${0}")/test-lulesh-ft.expected"
