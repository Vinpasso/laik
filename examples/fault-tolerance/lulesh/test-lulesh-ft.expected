Running problem size 10^3 per domain until completion
Num processors: 8
Total number of elements: 8000

To run other sizes, use -s <integer>.
To run a fixed number of iterations, use -i <integer>.
To run a more or less balanced region set, use -b <integer>.
To change the relative costs of regions, use -c <integer>.
To print out progress, use -p
To write an output file for VisIt, use -v
See help (-h) for more options

 
=== TESTS ON REPARTITIONING MODE === 
cycle = 1, time = 1.068124e-05, dt=1.068124e-05
cycle = 2, time = 2.349873e-05, dt=1.281749e-05
cycle = 3, time = 2.789208e-05, dt=4.393346e-06
cycle = 4, time = 3.156086e-05, dt=3.668785e-06
cycle = 5, time = 3.483144e-05, dt=3.270581e-06
cycle = 6, time = 3.786833e-05, dt=3.036883e-06
cycle = 7, time = 4.075562e-05, dt=2.887293e-06
cycle = 8, time = 4.354443e-05, dt=2.788812e-06
cycle = 9, time = 4.626937e-05, dt=2.724937e-06
cycle = 10, time = 4.895588e-05, dt=2.686510e-06
cycle = 11, time = 5.217969e-05, dt=3.223812e-06
cycle = 12, time = 5.581954e-05, dt=3.639850e-06
cycle = 13, time = 5.926037e-05, dt=3.440828e-06
cycle = 14, time = 6.250299e-05, dt=3.242623e-06
cycle = 15, time = 6.554721e-05, dt=3.044226e-06
cycle = 16, time = 6.845350e-05, dt=2.906290e-06
cycle = 17, time = 7.127318e-05, dt=2.819677e-06
cycle = 18, time = 7.405032e-05, dt=2.777138e-06
cycle = 19, time = 7.682702e-05, dt=2.776701e-06
cycle = 20, time = 7.960372e-05, dt=2.776701e-06
cycle = 21, time = 8.238042e-05, dt=2.776701e-06
cycle = 22, time = 8.515712e-05, dt=2.776701e-06
cycle = 23, time = 8.793383e-05, dt=2.776701e-06
cycle = 24, time = 9.100721e-05, dt=3.073382e-06
cycle = 25, time = 9.408059e-05, dt=3.073382e-06
cycle = 26, time = 9.715397e-05, dt=3.073382e-06
cycle = 27, time = 1.006571e-04, dt=3.503118e-06
cycle = 28, time = 1.041602e-04, dt=3.503118e-06
cycle = 29, time = 1.083436e-04, dt=4.183372e-06
cycle = 30, time = 1.130908e-04, dt=4.747189e-06
cycle = 31, time = 1.176481e-04, dt=4.557295e-06
cycle = 32, time = 1.220514e-04, dt=4.403381e-06
cycle = 33, time = 1.263491e-04, dt=4.297636e-06
cycle = 34, time = 1.305781e-04, dt=4.229043e-06
cycle = 35, time = 1.347762e-04, dt=4.198111e-06
Starting Repartitioning, current runtime = 0.000000 s
Previous Time per Iteration: 0.000000 s 
New world size: 1
Repartition Done in 0.051567 s on Rank 0
cycle = 31, time = 1.176481e-04, dt=4.557295e-06
cycle = 32, time = 1.220514e-04, dt=4.403381e-06
cycle = 33, time = 1.263491e-04, dt=4.297636e-06
cycle = 34, time = 1.305781e-04, dt=4.229043e-06
cycle = 35, time = 1.347762e-04, dt=4.198111e-06
cycle = 36, time = 1.389743e-04, dt=4.198111e-06
cycle = 37, time = 1.431725e-04, dt=4.198111e-06
cycle = 38, time = 1.473706e-04, dt=4.198111e-06
cycle = 39, time = 1.515687e-04, dt=4.198111e-06
cycle = 40, time = 1.562730e-04, dt=4.704301e-06
cycle = 41, time = 1.609773e-04, dt=4.704301e-06
cycle = 42, time = 1.656816e-04, dt=4.704301e-06
cycle = 43, time = 1.703859e-04, dt=4.704301e-06
cycle = 44, time = 1.750902e-04, dt=4.704301e-06
cycle = 45, time = 1.797569e-04, dt=4.666679e-06
cycle = 46, time = 1.843243e-04, dt=4.567459e-06
cycle = 47, time = 1.888070e-04, dt=4.482654e-06
cycle = 48, time = 1.932201e-04, dt=4.413125e-06
cycle = 49, time = 1.975783e-04, dt=4.358243e-06
cycle = 50, time = 2.018977e-04, dt=4.319329e-06
Run completed:  
   Problem size        =  20 
   MPI tasks           =  8 
   Iteration count     =  50 
   Final Origin Energy = 6.483837e+05 
   Testing Plane 0 of Energy Array on rank 0:
        MaxAbsDiff   = 8.731149e-11
        TotalAbsDiff = 1.037000e-10
        MaxRelDiff   = 7.684754e-13


Elapsed time         =       0.00 (s)
Grind time (us/z/c)  = 1.7383252e-309 (per dom)  (2.1729065e-310 overall)
FOM                  =        inf (z/s)

Time After Repartitioning: 0.000000, per Iteration 0.000000

